src/scoping-shim.js

```diff
@@ -239,6 +234,10 @@ export default class ScopingShim {
     if (!styleInfo) {
       styleInfo = this._prepareHost(host);
     }
+    // if there is no style info at this point, bail
```

Comment:
above `const  styleInfo = StyleInfo.get(host) || this._prepareHost(host);`

Reply:
Done